openpype/modules/deadline/plugins/publish/collect_default_deadline_server.py

```diff
@@ -32,22 +32,28 @@ def process(self, context):
 
         # get default deadline webservice url from deadline module
         self.log.debug(deadline_module.deadline_urls)
-        context.data["defaultDeadline"] = deadline_module.deadline_urls["default"]  # noqa: E501
+        default_deadline_webservice = deadline_module.deadline_urls["default"]
 
         context.data["deadlinePassMongoUrl"] = self.pass_mongo_url
 
         deadline_servers = (context.data
```

Comment:
Can we use AYON_SERVER_ENABLED? Easier to find out AYON vs. OpenPype differences in future.
```python
deadline_settings = context.data["project_settings"]["deadline"]
deadline_server_name = None
if AYON_SERVER_ENABLED:
    deadline_server_name = deadline_settings["deadline_server"]
else:
    deadline_servers = deadline_settings["deadline_servers"]
    if deadline_servers:
        deadline_server_name = deadline_servers[0]

Reply:
I agree, this is more straightforward